rabbitmq/publisher.go

```diff
@@ -163,6 +187,36 @@ func (rp *rabbitMqPublisher) publishFinalizedToExchange(finalizedBlock data.Fina
 	}
 }
 
+func (rp *rabbitMqPublisher) publishTxsToExchange(blockTxs data.BlockTxs) {
+	if rp.cfg.TxsEventsExchange != "" {
```

Comment:
refactor:
```
if rp.cfg.TxsEventsExchange == "" {
   return
}
// the rest
```

Reply:
i was thinking, it might be better to check these config vars in `checkArgs`

Reply:
refactored to check exchange names in constructor